src/oaklib/datamodels/ontology_metadata.yaml

```diff
@@ -720,15 +735,24 @@ slots:
     is_a: provenance_property
     slot_uri: dcterms:contributor
     multivalued: true
-    range: Thing
+    range: HomoSapiens
```

Comment:
I don't think this is the best way to provide a range for the slot. The contributor does not have to be human - it should be possible to express arbitrary agents here, in particular RORs, or wikidata identifiers for people that don't want an orcid..